published/024-2019-jan-8.md

```diff
@@ -0,0 +1,345 @@
+## Welcome to the first IPFS Weekly of 2019! 🎉
+
+For this first week back, we’ve put together a very special issue looking back on all that you, the [InterPlanetary File System (IPFS)](https://ipfs.io/) community, accomplished in the last year. 
+
+# 2018 was a year of significant milestones
+  
+**January:** Mozilla adds [support for Decentralization Protocols](https://blog.mozilla.org/addons/2018/01/26/extensions-firefox-59/) in popular web browser, Firefox.  
+  
+**February:** The world was introduced to [libp2p](https://blog.keep.network/introduction-to-libp2p-57ce6527babe), a peer-to-peer networking stack built modularly, with IPFS.  
+  
+**March:** [<code>js-ipfs v0.28.0</code>](https://github.com/ipfs/js-ipfs/issues/1228) was released, Juan Benet presented [Redecentralizing the Web](https://www.youtube.com/watch?v=eeN_d4MpnNI) to MIT’s largest research lab, the Computer Science & Artificial Intelligence Lab, and <code>go-ipfs</code> released [version 0.4.14](https://blog.ipfs.io/34-go-ipfs-0.4.14).  
+  
+**April:** We introduced [IPFS Companion 2.2.0](https://blog.ipfs.io/35-ipfs-companion-2-2-0/) and gave folks a look at [<code>go-ipfs</code> on Windows](https://blog.ipfs.io/36-a-look-at-windows/).  
+  
+**May:** We announced the first ever [IPFSConf](https://twitter.com/IPFSbot/status/998964366077816832), Textile introduced [a personal IPFS](https://medium.com/textileio/your-photos-decentralized-and-encrypted-a-first-look-at-the-tech-inside-textile-photos-9b0155c25f15) that runs on your mobile device for true decentralized storage. [<code>js-ipfs 0.29.0</code> was released](https://github.com/ipfs/js-ipfs/issues/1320) and we closed out the month by getting the [IPFS Cluster 0.4.0 release candidates](https://twitter.com/hecturchi/status/1001074944481120256) out (https://twitter.com/hecturchi/status/1001074944481120256) with a newly launched [IPFS Cluster website](https://cluster.ipfs.io/) to boot.  
+  
+**July:** The [Awesome IPFS list](https://github.com/ipfs/awesome-ipfs) on GitHub was [relaunched as a website](https://twitter.com/daviddias/status/1024281949416771584). Released [<code>js-ipfs v0.30.0</code>](https://github.com/ipfs/js-ipfs/issues/1375) and [v0.31.0](https://github.com/ipfs/js-ipfs/issues/1458) back to back. <code>js-libp2p</code> had its first [_official_ release] (https://github.com/libp2p/js-libp2p/issues/226), and lastly, [<code>go-ipfs v0.4.17</code>](https://twitter.com/Whyrusleeping/status/1023116433113247746) was released.  
+  
+**August:** We celebrated [Lab Day 2018](https://lab-day.com/2018/welcome) in San Francisco, California and released both the [InterPlanetary Linked Data (IPLD)](https://explore.ipld.io/) explorer site and the official site for <code>js-ipfs</code>: [js.ipfs.io](https://js.ipfs.io/). We closed out the summer with the release of [IPFS Cluster 0.5.0](https://cluster.ipfs.io/news/20180824_0.5.0_release/).  
```

Comment:
https://lab-day.com/2018/welcome returns error 404, perhaps we should change it to https://lab-day.com/aug-03-2018/ and also add link to [this youtube playlist?](https://www.youtube.com/playlist?list=PLhuBigpl7lqvIymGaM7A_VT4CYZW3R_4Q)